server/index.js

```diff
@@ -26,15 +26,23 @@ app.get('/reviews/:id', (req, res) => {
 
 const jsonParser = bodyParser.json();
 app.patch('/review', jsonParser, (req, res) => {
-  console.log(req.body.reviewId);
-  console.log(req.body.likes_count);
+  let newLikesCount;
+  if (req.body.likedStatus === 'like') {
+    newLikesCount = req.body.likesCount + 1;
+  } else {
+    newLikesCount = req.body.likesCount - 1;
+  }
+
   review.findOneAndUpdate(
```

Comment:
Small nitpick if you're refactoring later - this may be a better fit for a method stored in the `database` folder as a module that can then be used here. Would let you update likes from other spots in the app later.